pages/index.js

```diff
@@ -1,5 +1,65 @@
+import { BUILD_ID_FILE } from 'next/dist/shared/lib/constants';
```

Comment:
what does this do?🤔

Reply:
i don't know where it comes from. Maybe from the timer installation? Or an unnoticed auto import.

Reply:
that would also be my guess. I would remove the import and check whether your app still works just fine without it to see whether it is a necessary import or was just an unnoticed auto import